_ci/settings.xml

```diff
@@ -6,6 +6,15 @@
                 <activeByDefault>true</activeByDefault>
             </activation>
             <repositories>
+                <repository>
+					<id>central</id>
```

Comment:
Minor: Formatting looks a bit out of order.

Reply:
Fixed